.github/workflows/logging.yml

```diff
@@ -0,0 +1,108 @@
+name: logging
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - "sentry_logging/**"
+      - ".github/workflows/logging.yml"
+  pull_request:
+    paths:
+      - "sentry_logging/**"
+      - ".github/workflows/logging.yml"
```

Comment:
why do we need this?

Reply:
We don't neccessarily need this. I just thought it would be nice if this package only gets build if it's changed. It's very simple and I suspect most SDK changes won't affect this package anyway. 
It's your call if it should be included or not.